website/docs/getting-started-herotypes.md

```diff
@@ -0,0 +1,148 @@
+<h1 align="center">HeroTypes</h1>
+
+In Texthero, we're always working with Pandas Series and Pandas Dataframes to gain insights from text data! To make things easier and more intuitive, we differentiate between different types of Series/DataFrames, depending on where we are on the road to understanding our dataset.
+
+<h2 align="center">Overview</h2>
+
+When working with text data, it is easy to get overwhelmed by the many different functions that can be applied to the data. We want to make the whole journey as clear as possible. For example, when we start working with a new dataset, we usually want to do some preprocessing first. At the beginning, the data is in a DataFrame or Series where every document is one string. It might look like this:
+```python
+                                    text
+document_id                             
+0            "Text in the first document"
+1            "Text in the second document"
+2            "Text in the third document"
+3            "Text in the fourth document"
+4                                    ...
+
+```
+
+ Consequently, in Texthero's _preprocessing_ module, the functions usually take as input a Series where every cell is a string, and return as output a Series where every cell is a string. We will call this kind of Series _TextSeries_, so users know immediately what kind of Series the functions can work on. For example, you might see a function
+ ```python
+remove_punctuation(s: TextSeries) -> TextSeries
+ ```
+in the documentation. You then know that this function takes as input a _TextSeries_ and returns as output a _TextSeries_, so it can be used on a DataFrame or Series in the preprocessing phase of your work, where each document is one string.
```

Comment:
I don't think it can be used on a dataframe as well, just on the textcolumn, which will then be interpreted as a series. So this sentence might be a bit confusing

Reply:
yes, you're right - will change :running_man: